tests/functional/test_provider.py

```diff
@@ -397,3 +397,17 @@ def test_create_access_list(eth_tester_provider, vyper_contract_instance, owner)
     tx = vyper_contract_instance.setNumber.as_transaction(123, sender=owner)
     with pytest.raises(APINotImplementedError):
         eth_tester_provider.create_access_list(tx)
+
+
+def test_auto_mine(eth_tester_provider):
+    eth_tester_provider.auto_mine = False
+    assert not eth_tester_provider.auto_mine
+
+    # Ensure can still manually mine.
+    block = eth_tester_provider.get_block("latest").number
```

Comment:
attempted to repro https://github.com/ApeWorX/ape/issues/1969 but mining still seems to work, at least at the provider level.

Reply:
Is this a blocker?

Reply:
not for this feature particularly. if we can get them to upgrade we can see if that helps them else will have to look for more info.